src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,26 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+import usersFromServer from '../../api/users.json';
+
+export const PostInfo = ({ post }) => {
+  const user = usersFromServer.find(el => el.id === post.userId);
```

Comment:
Ensure that `usersFromServer` is an array of user objects with an `id` property. This line assumes that `post.userId` corresponds to a valid user in `usersFromServer`. If the data structure is different, this logic may need adjustment.